sros2/sros2/api/__init__.py

```diff
@@ -40,6 +40,11 @@
     load_policy,
 )
 
+try:
+    from rclpy.utilities import get_rmw_implementation_identifier
+except ImportError:
```

Comment:
If `rclpy` is not available.
It may not make much sense now, as `rclpy` is a required dependency.

I can delete that check for now, and it can be re-added in the future when getting rid of `rclpy` dependency.

Reply:
Seems sensible to let the error propagate if rclpy is a required dep.

Reply:
Done in https://github.com/ros2/sros2/pull/177/commits/51405d63f195e0f9917a687b0d2b13825c679018.